subprojects/language-groovy/src/integTest/groovy/org/gradle/language/groovy/GroovyCompileJavaVersionTrackingIntegrationTest.groovy

```diff
@@ -1,108 +0,0 @@
-/*
- * Copyright 2017 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.gradle.language.groovy
-
-import org.gradle.integtests.fixtures.AbstractIntegrationSpec
-import org.gradle.integtests.fixtures.AvailableJavaHomes
-import org.gradle.internal.jvm.Jvm
-import org.gradle.util.Requires
-import org.gradle.util.internal.TextUtil
-
-import static org.gradle.api.JavaVersion.VERSION_1_8
-import static org.gradle.api.JavaVersion.VERSION_11
-
-@Requires(adhoc = { AvailableJavaHomes.getJdk(VERSION_1_8) && AvailableJavaHomes.getJdk(VERSION_11) })
-class GroovyCompileJavaVersionTrackingIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
These tests are replaced with `up-to-date depends on the toolchain` test.

Also, one of these removed tests does not seem to be valid, as it modifies the build script in the second `when` clause and proceeds to assert non-up-to-dateness.